tests/testfiles/CommonMark_0.30/README.md

```diff
@@ -1,2 +1,4 @@
 This directory contains test files that were taken from the
-CommonMark project specification.
```

Comment:
Here, we would like to also link the C# tool that we used to semi-automatically produce the test files.